### DISCUSSION TITLE
Spring Boot Metric In Kubernetes Cluster

### DISCUSSION THREAD

**Discussion by heriparid** [SigNoz/signoz#1415] [Q&A]
Hi,
Thanks for providing this such great Platform for APM.
So far, I have successfully integrated Spring Boot tracer and export to SigNoz' by using OpenTelemetry protocol. Thanks to Sleuth OTLP library. My setup so far:

1. Installing signoz in Kubernetes using helm
2. Export trace using sleuth library, by specifying SigNoz Otel endpoint
`spring.sleuth.otel.exporter.otlp.endpoint=http://signoz-otel-collector.<namespace>.svc.cluster.local:4317`

Having said that, I need a guidance on how signoz collecting Spring Boot metric which generated by micrometer-registry-prometheus in Kubernetes Cluster
By using micrometer-registry-prometheus, my application exposes the metrics with the this endpoint /actuator/prometheus.

In my Kubernetes Cluster there're some Installed ConfigMaps one of them i.e `signoz-otel-collector-metrics`. Do I have to manually edit the `signoz-otel-collector-metrics` config map on the following screen shot image:
![image](https://user-images.githubusercontent.com/3928060/179400178-f4ac8f55-3b33-4ec5-994a-e9b1ae6b78a6.png)

**Comment by srikanthccv** [comment]
@heriparid Yes, you have to add the new job to scrape configs list for SigNoz to collector metrics exported by Prometheus clients.
